Slim snippet (#54)
tskit-dev/tskit-site: _software/SLiM.md

category: simulate
permalink: /SLiM
code_snippet: |2
  initialize() {
      initializeTreeSeq();
      initializeMutationRate(1e-8);
      initializeMutationType("m1", 0.5, "e", 0.001);
      initializeGenomicElementType("g1", m1, 1.0);
      initializeGenomicElement(g1, 0, 999999);
      initializeRecombinationRate(1e-8);
  }
  1 { sim.addSubpop("p1", 500); }
  2000 late() { sim.treeSeqOutput("out.trees"); }
---

> SLiM is an evolutionary simulation framework that combines a powerful engine
> for population genetic simulations with the capability of modeling arbitrarily
> complex evolutionary scenarios. The underlying individual-based simulation

Reviewer: ```suggestion
SLiM is a forwards-time population genetics simulator capable of simulating arbitrarily
complex evolutionary scenarios. The underlying individual-based simulation
```

engine is highly optimized to enable modeling of entire chromosomes in

> large populations. There is also a graphical user interface on macOS and Linux

Reviewer: ```suggestion
large populations. There is also a graphical user interface
```
People run it on Windows also...

Author: True - this is simply copied straight off the main SLiM website - you might want to point out to Ben that he can change that there too?
